### PR TITLE
mad lib function version

### PR DESCRIPTION
**Pull request by Rosebeatty**

### CODE REVIEW

**Comment by fabientownsend** [comment]
There is a way to improve your code style by following some conventions, here is the one that I usually use which is written by Airbnb: https://github.com/airbnb/ruby

That could help to make your code easier to read

File: exercise_name4.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+more = true
+while more==true
+
+def my_method(word)
```
Comment (fabientownsend) [inline exercise_name4.rb]: I think you can find a better name than `my_method`.

An approach for that is to try to code by intention. How would you communicate your intention through the code? Would it be comprehensible to someone that doesn't know anything about coding?

For example:
``` ruby
my_method("verb")
```
could be something like
``` ruby
ask_user_a("verb")
```

Comment (Rosebeatty) [reply exercise_name4.rb]: Agreed, I'll change it to something more explicit!

File: exercise_name4.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+play_again = true
+
+while play_again == true
+  def ask_user(word)
+     puts "Enter a" + word
+  return gets.chomp
+end
```
Comment (fabientownsend) [inline exercise_name4.rb]: Is this end for while or def? :)

Comment (Rosebeatty) [reply exercise_name4.rb]: def, i'll indent it :)

**Pull request merged**